### PR TITLE
Add slightly kinder database url defaults

### PR DESCRIPTION
**Pull request by pauldambra**
## Changes

builds on top of #7675 

But reads PGHOST, PGUSER, PGPASSWORD, and PGPORT environment variables in case people already have those set to working values for them

## How did you test this code?

as in #7675

### CODE REVIEW

File: bin/tests [hunk 1]
```diff
@@ -23,5 +23,10 @@ if [ $# -eq 0 ]; then
 fi
 
 export REDIS_URL='redis:///'
-psql posthog -d postgres://posthog:posthog@localhost:5432 -c "drop database if exists test_posthog"
+
+PG_HOST="${PGHOST:=localhost}"
+PG_USER="${PGUSER:=posthog}"
+PG_PASSWORD="${PGPASSWORD:=posthog}"
+PG_PORT="${PGPORT:=5432}"
+psql posthog -d "postgres://${PG_USER}:${PG_PASSWORD}@${PG_HOST}:${PG_PORT}" -c "drop database if exists test_posthog"
```
Comment (hazzadous) [inline bin/tests:31]: psql will use the env vars right? So there's no need to specify the connection string.

Comment (hazzadous) [reply bin/tests:31]: Actually you'd want to remove the `_` in the env names for that do work

Comment (pauldambra) [reply bin/tests:31]: It is only setting local variables not environment variables. So, it respects when someone has the env vars set. But someone who doesn't (me :) or a new contributor) gets sensible defaults

Comment (pauldambra) [reply bin/tests]: better like this? https://github.com/PostHog/posthog/pull/7730/commits/4991f28808bf81cb5d4f9b21e41ddb6e773bbd21

Comment (hazzadous) [reply bin/tests]: > It is only setting local variables not environment variables

Yeah maybe that's nicer actually 😳  I like your way.

**Comment by pauldambra** [comment]
I'm going to bias towards action and merge it

**Pull request merged**